### PR TITLE
[DO NOT MERGE] Document Jobs API

### PR DESCRIPTION
**Pull request by despo**
Concerns: A number of field definitions have diverted from the JobPosting standard or don't match the existing definitions.

Suggestions have been included of complimentary fields that could be used to capture certain information.

### CODE REVIEW

File: Jobs_API.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    "identifier": "137903"
+  },
+  "validThrough": "2018-05-21T00:00:00+00:00",
+  "workHours": ""
+}
+```
+
+### Concerns
+
+The following captured fields are not made available via JSON as they don't map to the JobPosting schema definitino.
```
Comment (gazaston) [inline Jobs_API.md]: Typo ('definition')

**Comment by tahb** [comment]
We recently spoke to GDS and their feedback on documentation was to [use the Open API v3 standard](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md). Did you consider this as an option or foresee any issues with trying to fit our documentation into it?

I think we should remember to document how our search API and what the state of jobs (expired, draft, etc) that will be returned by default.

**Comment by despo** [comment]
@tahb this is not trying to satisfy the task of documenting or integrating a documenting standard, for now just trying to resolve fundamental issues with mismatching definitions for the chat tomorrow and this seemed to be the best way to get through to everyone the state that we are at.

**Comment by leannedxw** [comment]
I have had a look and have some general comments which are listed below. I have also just read the comment history and in particular note that this initial version has been pulled together just to progress the mismatching definitions issues @despo . Many of my comments are more generally applicable to our target state documentation, but I've included them here anyway for future consideration:

- In general regarding the discrepancies, are there any of these that we want to raise as issues to the open standard community? We can do so here: [https://github.com/schemaorg/schemaorg/issues?utf8=%E2%9C%93&q=is%3Aissue+JobPosting] Potential things we might want to raise:

>Description and *responsibilities  - duplication, could these be merged?
>JobBenefits and *incentivecompensation - again could these be merged?
>Could start and end date parameters be added to the jobposting schema? Having a quick look through the issues log, can see that the start date issue has been raised by at least one other in the community already 

- Is there any reason why we can’t make all fields available via the API, including the ones that don’t map to the schema? Sorry if this is a dumb question and this is because of a technical constraint, I’m just thinking about maximising the usability of pulled listings for job-seekers. If it is validated information that our job-seekers need when using Teaching Jobs, then they would need the same information when looking at a third party site. What is our justification for removing?

- Can the GitHub repository be renamed to Teaching Jobs?

- I had a look at the notify service API documentation (e.g. [https://github.com/alphagov/notifications-ruby-client] ) and it prompted a few ideas on formatting etc:

> Could we have an intro statement along lines of  'This documentation is for developers interested in integrating their service with Teaching Jobs’
> Could we have a hyperlinked table of contents to improve navigability? 
> The ">Click here to expand for more information." Links are also quite useful for navigability

**Comment by tahb** [comment]
Notes on what's still to do https://gist.github.com/despo/203c402a76c48ee4bf8176e5d3fc2251

**Comment by lozette** [comment]
Updated PR title to exclude it from the Seal

**Pull request closed without merging**